src/frontend/src/catalog/system_catalog/rw_catalog/rw_relation_info.rs

```diff
@@ -12,20 +12,161 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use risingwave_common::types::DataType;
-
-use crate::catalog::system_catalog::SystemCatalogColumnsDef;
-
-pub const RW_RELATION_INFO_TABLE_NAME: &str = "rw_relation_info";
-
-pub const RW_RELATION_INFO_COLUMNS: &[SystemCatalogColumnsDef<'_>] = &[
-    (DataType::Varchar, "schemaname"),
-    (DataType::Varchar, "relationname"),
-    (DataType::Int32, "relationowner"),
-    (DataType::Varchar, "definition"),
-    (DataType::Varchar, "relationtype"),
-    (DataType::Int32, "relationid"),
-    (DataType::Varchar, "relationtimezone"), /* The timezone used to interpret ambiguous
-                                              * dates/timestamps as tstz */
-    (DataType::Varchar, "fragments"), // fragments is json encoded fragment infos.
-];
+use std::sync::LazyLock;
+
+use risingwave_common::catalog::RW_CATALOG_SCHEMA_NAME;
+use risingwave_common::error::Result;
+use risingwave_common::row::OwnedRow;
+use risingwave_common::types::{DataType, ScalarImpl};
+use serde_json::json;
+
+use crate::catalog::system_catalog::{BuiltinTable, SysCatalogReaderImpl};
+
+// TODO: `rw_relation_info` contains some extra streaming meta info that's only meaningful for
+// streaming jobs, we'd better query relation infos from `rw_relations` and move these streaming
+// infos into anther system table.
+pub static RW_RELATION_INFO: LazyLock<BuiltinTable> = LazyLock::new(|| BuiltinTable {
```

Comment:
We'd better deprecate it after cloud migrate its reference to `rw_materialized_views` and add streaming graph info in it or add a new system table for streaming graph.